src/main/java/org/disq_bio/disq/serializer/DisqKryoRegistrator.java

```diff
@@ -125,6 +125,7 @@ public void registerClasses(final Kryo kryo) {
     kryo.register(htsjdk.variant.variantcontext.VariantContext.Type.class);
 
     // htsjdk.variant.vcf
+    kryo.register(htsjdk.variant.vcf.VCFAltHeaderLine.class);
```

Comment:
@tomwhite The htsjdk PR that introduced this new header line class also added a couple of others that should probably be registered for serialization as well: VCFSampleHeaderLine, VCFMetaHeaderLine, VCFPedigreeHeaderLine.

Reply:
Are those VCF header line changes binary compatible with htsjdk 2.18.x?  At some point we need to start thinking about how htsjdk and Disq releases synchronize with each other.

Reply:
@huermh Yes, the new header line classes are VCFHeaderLine-derived, so they should be binary compatible with previous VCFHeader/VCFHeaderLine consumers.